Cargo.toml

```diff
@@ -26,7 +26,7 @@ argon2 = "0.5"
 chrono = { version = "0.4", default-features = false, features = ["serde"] }
 validator = "0.18"
 cookie = "0.18"
-#ft-sdk = { version = "0.1.11", path = "../../ft-sdk/ft-sdk", features = ["sqlite-default", "auth-provider", "field-extractors"] }
-ft-sdk = { version = "0.1.14", features = ["sqlite-default", "auth-provider", "field-extractors"] }
+# ft-sdk = { path = "../ft-sdk/ft-sdk", features = ["sqlite-default", "auth-provider", "field-extractors"] }
+ft-sdk = { version = "0.1", features = ["sqlite-default", "auth-provider", "field-extractors"] }
```

Comment:
I actually prefer the more precise version numbering of our crates.

Reply:
Thanks. done in https://github.com/fastn-community/auth/pull/11/commits/7a284a269554b3e6d2dee5d0082b7cd98b15326e